src/components/headline.tsx

```diff
@@ -3,14 +3,14 @@
 import React, { ReactElement } from 'react';
 import { css, SerializedStyles } from '@emotion/core';
 import { headline, textSans } from '@guardian/src-foundations/typography';
-import { background, neutral, text } from '@guardian/src-foundations/palette';
 import { remSpace } from '@guardian/src-foundations';
+import { Format } from '@guardian/types/Format';
 
 import { Item } from 'item';
-import { darkModeCss as darkMode } from 'styles';
-import { getPillarStyles, PillarStyles } from 'pillarStyles';
 import StarRating from './starRating';
 import { Display, Design } from 'format';
+import { border } from 'editorialPalette';
```

Comment:
Do you intend to move `border` to `editorialStyles` also?

Reply:
I wasn't planning to _move_ it, but we could generate CSS declarations for it there as well. Would have to be `border-color` specifically rather than `border` I think?

Now that you've flagged this it occurs to me we could also name it something more generic in the `palette` module, like `line`?

Reply:
> Would have to be border-color specifically rather than border I think

Yeah, maybe. I was gonna suggest dropping `colour` or `color` from names because colour is implied by the fact it's in the editorial palette. It's up to you though, maybe there will be non-colour tokens here also (spacing, border radiuses)? In which case, specific is better.

> we could also name it something more generic in the palette module, like line

I think there's a distinction between `border` and `line`. Borders border elements, lines divide elements. Borders are encapsulated within a component, lines are defined at the layout level.

Reply:
> I was gonna suggest dropping `colour` or `color` from names because colour is implied by the fact it's in the editorial palette

Sorry, I was probably unclear there. I was thinking something along these lines:

1. The `editorialPalette` module would contain a colour token, called either `border` or `line`, which is _just_ a hex colour.

2. The `editorialStyles` module could contain a CSS declaration of the form:

```ts
const border = css`
  border-color: ${palette.border};
`;
```

When I said `border-color` I meant the CSS property rather than the name for the variable in our code 🙂. I think you're right, this is _just_ for colours.

> Borders border elements, lines divide elements

Ok, we have a semantic distinction between them so we should provide different names to avoid confusion, sounds good 👍.

Reply:
Right, yeah, me got confused there! Only 3 cups of tea this morning. You makes a lot of sense 👍